docs/resources/organization_settings.md

```diff
@@ -25,3 +25,5 @@ Organization settings can be imported by passing the organization slug to the im
 ```
 $ terraform import buildkite_organization_settings.test_settings test_org
 ```
+
+Your organization's slug can be found in your organisation's [settings](https://buildkite.com/organizations/settings) page.
```

Comment:
I think use this dynamic URL: `https://buildkite.com/organizations/~/settings` which redirects to the active org for the user

Reply:
Actually, we should probably improve the code behind this so that anything can be passed as the slug. That's because, to properly run any terraform command, we need the org slug to be configured for the provider anyway so its redundant. The problem is though that `terraform import` requires an ID to be passed.

🤷‍♂️ I guess it doesn't really matter requiring them to pass the slug, but something that does get at me is that almost everything else is graphql ID and I think that should be consistent. That is definitely out of scope here though

Reply:
That is a fair call - I've put the dynamic URL into the latest commit but worth assessing 👍